packages/teuchos/numerics/src/Teuchos_SerialDenseHelpers.hpp

```diff
@@ -182,6 +185,34 @@ bool setCol( const SerialDenseVector<OrdinalType, ScalarType>& v,
   return true;
 }
 
+/*! \related SerialDenseMatrix
+  \brief A templated, non-member, helper function for generating a random SerialDenseMatrix that is synchronized in parallel.
+
+  \param A - [in/out] SerialDenseMatrix to be filled with random numbers that are the same across processors
+*/
+template <typename OrdinalType, typename ScalarType>
+void randomSyncedMatrix( Teuchos::SerialDenseMatrix<OrdinalType, ScalarType>& A )
```

Comment:
Why not have this function take a `const Teuchos::Comm<int>&`?  It seems like a logical generalization.  On the other hand, I don't want to block this fix.